internal/resources/webform/webform-template.html

```diff
@@ -49,6 +49,7 @@ <h3>Response Headers</h3>
       <h3>Response Data</h3>
       <div id="grpc-response-req-stats"></div>
       <div id="grpc-response-data"></div>
+
```

Comment:
nit: let's undo this whitespace-only change

Reply:
done!